packages/oui-status/_mixins.less

```diff
@@ -0,0 +1,59 @@
+@import '../oui-color/_variables';
+@import './_variables';
+
+#oui {
+  .status-base(
+    @font-size: @oui-status-font-size,
```

Comment:
New rules! 🎉
Use directly the variable in body without arguments. KISS is the new watchword on this project.